### DISCUSSION TITLE
New Projects are not shown in Issue sidebar any more

### DISCUSSION THREAD

**Discussion by janpio** [community/community#38850] [Projects and Issues]
### Select Topic Area

Bug

### Body

At Prisma we are big users of GitHub Issues and also the new GitHub Projects (boards). Until recently, when an issue is part of a Project, that information (and an ability to edit that, and the fields of the board connected to it) were displayed in the right sidebar of the issue. Unfortunately, that functionality has disappeared.

2 examples (public and private repositories):
https://github.com/prisma/prisma/issues/15616
https://github.com/prisma/client-planning/issues/117

<img width="334" alt="image" src="https://user-images.githubusercontent.com/183673/201310875-e240b8da-c76d-4aca-9630-be4c89ee856a.png">
<img width="361" alt="image" src="https://user-images.githubusercontent.com/183673/201310927-6944c367-9b45-4381-a64c-431bda1ce100.png">

<img width="607" alt="image" src="https://user-images.githubusercontent.com/183673/201310988-42b72512-63d4-4d2f-a2ad-cf87c744208a.png">
<img width="602" alt="image" src="https://user-images.githubusercontent.com/183673/201311051-a952cd57-3548-40a3-b84f-e990c74d3828.png">

It would be great if the Project would be displayed in GitHub Issues sidebar again. Otherwise updating properties of issues in these projects is almost impossible.

PS: "Classis", old style Kamban board, are displayed as they were before.

**Comment by janpio** [comment]
Just noticed something that might be relevant here: Both projects board are private to our organization. Will test if that has impact.

**Reply by janpio** [reply]
It does not 😢 

https://github.com/prisma/prisma/issues/15616 is now part of https://github.com/orgs/prisma/projects/90/views/1 but the issue sidebar does not show that: 
<img width="431" alt="image" src="https://user-images.githubusercontent.com/183673/201311711-61b644ba-ff11-45fe-939b-a7be55407caa.png">
<img width="386" alt="image" src="https://user-images.githubusercontent.com/183673/201311795-eb735378-4a42-41fc-9c76-cbd96e3fa88e.png">

**Comment by theerebuss** [comment]
Hey @janpio 👋🏻 This seems like a possible scenario should the `Projects` setting have been disabled for the repository in question. Can you validate that these settings are both checked?
![checkboxes for Projects and Projects (classic) are shown in the settings page](https://user-images.githubusercontent.com/39736248/202179380-6edb35a5-74a7-470b-ab72-8f44b453aed3.png)
You should see these in in the `General` menu, of the repository's settings, by scrolling down to the `Features` section.
If that is not the case, we would appreciate if you could create a support ticket for us to track this and look further into it. You can do that on https://support.github.com/request

**Reply by EtienneM** [reply]
I do not have both options "Projects" and "Projects (classic)" in my repository settings. I only have "Projects". But indeed, checking "Projects" solves the issue 👍🏾

**Reply by Jolg42** [reply]
After checking the checkbox for https://github.com/prisma/prisma, I can now see it for https://github.com/prisma/prisma/issues/15616

So thank you @andrefcdias! 🎊
 
But also quite confusing that this checkbox influences the "Projects" sidebar box.
And even more confusing that we can see it selected when clicking on the cog. 😕

**Reply by janpio** [reply]
We also only have "Projects": 
<img width="599" alt="image" src="https://user-images.githubusercontent.com/183673/202206785-a9ee2bbc-052d-4c1f-bcdd-e5666387c13b.png">

This is surprising to me as the Projects I am talking about are actually organization-level projects. We disabled "Projects" in the repository as we do not want to have any repo level Projects there and did not anticipate this to impact the issue sidebar. 

(Additionally confusing is that this only impacted org level _new_ projects, the old/classic ones were not impacted at all.)

I think that confusion indicates that this is a bug and should be cleaned up somehow.

**Reply by janpio** [reply]
I had already created a support case, #1875279. 
I will share the update of this discussion there as well to make sure there is no duplicate work.

**Reply by EtienneM** [reply]
Same as Jolg42 and janplo, it's really confusing... :/

**Reply by theerebuss** [reply]
Indeed it is not clear since the classic projects were tied directly to repositories and the new ones are tied to users/organisations. This setting is intended to disable the assignment of issues to projects (and vice-versa) altogether for issues in that specific repository.
We have already identified these discrepancies, like the assignment of issues from the project side regardless of this setting, and are already tracking the issue.
And thank you for all the great feedback 🙏🏻

**Comment by JFWooten4** [comment]
+1. This happened in our :Org issue when a contributor with admin access added the item into their personal project. It shows up in the tracking log of items, but not the sidebar.
